crates/red_knot_python_semantic/resources/mdtest/literal/ellipsis.md

```diff
@@ -0,0 +1,7 @@
+# Ellipsis literals
+
+## Simple
+
+```py
+reveal_type(...)  # revealed: EllipsisType | ellipsis
+```
```

Comment:
The TODO here got lost, but it's fine; I'll address this in the statically-known branches PR.